index.html

```diff
@@ -841,9 +841,9 @@
               </li>
               <li>Determine which <a>payment handlers</a> support
               <var>identifier</var>. For each resulting payment handler, if
-              payment method specific capabilities supplied by the payment
-              handler match those provided by <var>data</var>, the payment
-              handler matches.
+              <var>data</var> contains filters that exclude the payment
```

Comment:
@adrianhopebailie, does this version work for you: "Determine which payment handlers support identifier. For each resulting payment handler, if payment method specific capabilities supplied by the payment handler do not match those provided by data, the payment handler does not match, otherwise the payment handler matches."